404.html

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: Page not found
+title: 404
```

Comment:
```suggestion
title: 404 - Page not found
```

Reply:
The title `404` is as illustrated in the docs at https://pmarsceill.github.io/just-the-docs/docs/navigation-structure/#excluding-pages. The suggested change would make it a string, which doesn't provoke the build failure in 0.3.1.